### PR TITLE
acpanel: linter warning fix

### PR DESCRIPTION
**Pull request by ricksterhd123**
Fixed linter warnings for acpanel, not tested in-game yet.

### CODE REVIEW

**Comment by Disinterpreter** [comment]
```
label1
label2
label3
```
Someone in the past didn't think about variables' name, should we set the another names?

**Comment by iDannz1** [comment]
🤗 Well, thank you for this PR, but im not sure if we should start fixing lint warnings before we decide 100% about which rules we will use.... (see #365)

**Comment by ricksterhd123** [comment]
I had that in mind, I thought it was be good used as reference until it was figured out. The issues in here were mainly unused variables or redeclaring previously declared variables.

Ricky
________________________________

**Comment by ricksterhd123** [comment]
Proceed however you wish, it was originally a draft. If you feel it should be closed until #365 is resolved then go ahead. I'll be back ;)

Ricky
________________________________

**Comment by iDannz1** [comment]
I mean, there are some chances we will end up removing or adding some new rule, and that might make you have to go back and re-edit this PR, but anyway, i believe the final answer should be yours, if you dont see any problem with that, i dont think you need to close, your time is yours 😁

**Comment by Dutchman101** [comment]
@iDannz1 principally there is no problem with having PR's like this one, #365 is only meant to optimize the workflow against false positives _for ourselves_ (when it come to the linting when PR's with new features are submitted) and not to set any sort of style/coding guidelines

**Comment by patrikjuvonen** [comment]
I think in general we want to apply automatic formatting and linting fixes, not focus on that stuff too much manually.

**Comment by Dutchman101** [comment]
> I think in general we want to apply automatic formatting and linting fixes, not focus on that stuff too much manually.

yeah, that's also a good idea to look into as soon the linter config has been optimized for MTA's Lua stack (so that it only fishes out things with a high probability to indicate a real flaw or subpar coding conventions used), of course depending on how much we can adapt this linter for purposes like that and maybe also coding style enforcement once we got our guidelines in order as well. I will copy this feedback over.

**Comment by ricksterhd123** [comment]
All the issues within this resource are mainly due to bad coding practices. I would argue that unused index in for loop could be ignored based on preference.

Ricky
________________________________

**Comment by jlillis** [comment]
I think this sort of PR should wait until after discussion in #365 is concluded.

File: [admin]/acpanel/s_img_mod.lua [hunk 1]
```diff
@@ -45,13 +45,13 @@ addEventHandler ( "onResourceStart", resourceRoot,
 function isImgModBlocked( name )
 	local defText = getBlockDefText()
 
-	local type = getPanelSetting( "blockmods.type" )
+	getPanelSetting( "blockmods.type" )
```
Comment (ArranTuna) [inline [admin]/acpanel/s_img_mod.lua:48]: If type was unused then surely this function call is redundant?

**Comment by ArranTuna** [comment]
These changes have already been fixed or the warnings suppressed.

**Pull request closed without merging**